MapboxCoreNavigationTests/MapboxCoreNavigationTests.swift

```diff
@@ -6,8 +6,8 @@ import TestHelper
 
 let response = Fixture.JSONFromFileNamed(name: "routeWithInstructions")
 let jsonRoute = (response["routes"] as! [AnyObject]).first as! [String : Any]
-let waypoint1 = Waypoint(coordinate: CLLocationCoordinate2D(latitude: 37.795042, longitude: -122.413165))
-let waypoint2 = Waypoint(coordinate: CLLocationCoordinate2D(latitude: 37.7727, longitude: -122.433378))
+let waypoint1 = Fixture.waypoint1
```

Comment:
nit: There is a `Fixture.route(from:)` that will parse a route including waypoints, no hard-coded waypoints are needed.

Reply:
Resolved ✅